internal/commands/initialize/initialize.go

```diff
@@ -232,3 +241,90 @@ func internalTemplates() {
 	pterm.DefaultTable.WithHasHeader().WithData(internal).Render()
 
 }
+
+func writeConfig(path string, existsValues map[string]string) error {
+	allSettings := schema.GetJsonSchema()
+
+	var str string
+
+	str += getProperties(allSettings, existsValues)
+
+	f, err := os.Create(path)
+	if err != nil {
+		return err
+	}
+
+	defer f.Close()
+
+	_, err = f.WriteString(str)
+	if err != nil {
+		return err
+	}
+
+	return err
+}
+
+func getProperties(settings interface{}, existsValues map[string]string) string {
+	var strRoot string
+	var strBlocks string
+
+	properties, ok := settings.(map[string]interface{})["properties"]
+	if ok {
+		propertiesMap, ok := properties.(map[string]interface{})
+		if ok {
+			for pn, pv := range propertiesMap {
+				pm, ok := pv.(map[string]interface{})
+				if ok {
+					_, ok := pm["deprecationMessage"]
+					if ok {
+						continue
+					}
+					_, ok = pm["patternProperties"].(map[string]interface{})
+					if ok {
+						pd, ok := settings.(map[string]interface{})["definitions"].(map[string]interface{})[pn]
+						desc := ""
+						if ok {
+							strBlocks += fmt.Sprintf("\n# [%s.<name>]%s\n", pn, desc)
+							strBlocks += getProperties(pd, map[string]string{})
+						}
+					}
+					pt, ok := pm["type"].(string)
+					if !ok {
+						pt = "boolean"
+					}
+					pd := pm["description"].(string)
+					switch pt {
+					case "string":
+						v, ok := existsValues[pn]
+						if ok && pn != "env" {
+							strRoot += fmt.Sprintf("%-36s\t#%s\n", fmt.Sprintf("%s = \"%s\"", pn, v), pd)
```

Comment:
Is there any specific reason why you haven't accepted the spacing after the hash sign? Per TOML that should be the standard. So like:
```
# home = ""                              # (optional) User home directory can be specified here. Normally $HOME is used.
```
and not 
```
#home = ""                              #(optional) User home directory can be specified here. Normally $HOME is used.
```